gradle/testing/defaults-tests.gradle

```
@@ -186,16 +188,29 @@ allprojects {
 class LoggingFileArgumentProvider implements CommandLineArgumentProvider {
   @InputFile
   @PathSensitive(PathSensitivity.RELATIVE)
-  File loggingConfigFile
+  private final RegularFileProperty loggingConfigFile
```

the `private` keyword has no value in groovy. i would remove it.

```suggestion
  RegularFileProperty loggingConfigFile
```